src/v2/Apps/Artist/Routes/Overview/Components/ArtistArtworkFilter.tsx

```diff
@@ -40,6 +44,15 @@ const ArtistArtworkFilter: React.FC<ArtistArtworkFilterProps> = props => {
       aggregations={sidebarAggregations.aggregations as any}
       counts={artist.counts}
       onChange={updateUrl}
+      onArtworkBrickClick={artwork => {
+        tracking.trackEvent(
+          clickedMainArtworkGrid({
+            contextPageOwnerType: OwnerType.artist,
```

Comment:
Should this be using `PageOwnerType`?

Reply:
This is correct -- we only use `PageOwnerType` on the schema end.

Reply:
We should include optional properties `contextPageOwnerSlug` and `contextPageOwnerId` here.